js/script.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-env browser */
+/* eslint no-unused-vars: [1, {"argsIgnorePattern": "[evt]"}] */
+
+const gameBoard = (() => {
+  let board = ['1', '4', '7', '2', '5', '8', '3', '6', '9'];
+
+  const addPosition = (position, mark) => {
+    board[position - 1] = mark;
+  };
+
+  const resetBoard = () => {
+    board = ['1', '4', '7', '2', '5', '8', '3', '6', '9'];
+    return board;
+  };
+  return {
+    board,
+    addPosition,
+    resetBoard,
+  };
+})();
```

Comment:
As I have summarized in my first point of the general review, this should be in a file of its own. You can then import it into any file you want to call it from.